.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,18 @@
+name: Trivy
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
+  pull_request:
+  merge_group:
+  workflow_dispatch:
```

Comment:
Do we need workflow dispatch here?  I understand the other 3, but I'm not sure what the motivation is for this trigger.

Reply:
Cargo culting; I suppose it doesn't make much sense here.